testing/iwd/APKBUILD

```diff
@@ -0,0 +1,35 @@
+pkgname=iwd
+pkgver=0.12
+pkgrel=0
+pkgdesc="Wireless daemon for Linux"
+url="https://git.kernel.org/pub/scm/network/wireless/iwd.git"
+arch="all"
+license="GPL-2.0"
+makedepends="dbus-dev libell-dev linux-headers readline-dev"
+source="https://cdn.kernel.org/pub/linux/network/wireless/iwd-${pkgver}.tar.xz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	./configure \
+		--prefix=/usr \
+		--disable-systemd-service \
+		--enable-external-ell
+}
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	make check
+}
+
+package() {
+	cd "$builddir"
+	make install
```

Comment:
I think this is the reason for the build error (permission denied).
I think this line should read like:
```make DESTDIR="$pkgdir" install```